modules/ding_permissions/ding_permissions.module

```diff
@@ -1000,12 +1001,10 @@ function ding_permissions_secure_permissions($role) {
  * Implements hook_preprocess_HOOK().
  */
 function ding_permissions_preprocess_html(&$variables) {
-  global $user;
```

Comment:
Updated the PR with the proposed solution plus an additional safety measure.

Tested and works on all roles from administrators to guest_blogger. User 1 still sees two bars though, but I guess that doesn't matter much?